openadmet/models/features/gat_featurizer.py

```diff
@@ -0,0 +1,175 @@
+from typing import Optional, List, Any
+import torch
+from torch_geometric.data import Data
+from torch_geometric.loader import DataLoader
+from loguru import logger
+
+try:
```

Comment:
Should just be an error if no `rdkit`. I wouldn't even bother with this guard, just let it fail with `ImportError` for rdkit.